Gifski/Components/TrimmingAVPlayer.swift

```diff
@@ -284,6 +283,31 @@ final class TrimmingAVPlayerViewController: NSViewController {
 			}
 			.store(in: &cancellables)
 	}
+	func onNewDurationRange(durationRange newItemDurationRange: ClosedRange<Double>?) {
+		guard let newItemDurationRange else {
+			currentItemDurationRange = nil
+			return
+		}
+		defer {
+			currentItemDurationRange = newItemDurationRange
+		}
+		guard let timeRange,
+			  let currentItemDurationRange else {
```

Comment:
Code style. Applies everywhere.
```suggestion
		guard
			let timeRange,
			let currentItemDurationRange
		else {
```

Reply:
Sorry about this one. I was a bit confused because the alignment did look good on both, xcode and vscode (see below). But, I turned on whitespace and it looks like it was a interplay between tabs and spaces that the Github file viewer doesn't like. I'll switch everything over to pure tabs.

<img width="780" height="245" alt="Screenshot 2025-10-04 at 12 26 51 PM" src="https://github.com/user-attachments/assets/92e99edb-5400-4c62-8261-5096a0dfbd4e" />
<img width="466" height="183" alt="Screenshot 2025-10-04 at 12 29 07 PM" src="https://github.com/user-attachments/assets/d629b376-b7c4-47f5-9f8c-7992c122ca3e" />